modules/programs/vivid.nix

```
@@ -0,0 +1,141 @@
+{ config, lib, pkgs, ... }:
+let
+  yaml = pkgs.formats.yaml { };
+  bashLine = theme:
+    ''export LS_COLORS="$(${lib.getExe pkgs.vivid} generate ${theme})"'';
+  fishLine = theme:
+    "set -Ux LS_COLORS $(${lib.getExe pkgs.vivid} generate ${theme})";
```

```suggestion
    "set -gx LS_COLORS (${lib.getExe pkgs.vivid} generate ${theme})";
```

I'd prefer `set -gx`, as the line is loaded on each shell session anyway. Also, please remove the `$` sign, as it's not "native" fish syntax (it's supported only for compatibility).

---

Thanks for the heads up! I changed the flag and removed the `$` sign before the command.